Sources/JWT/SignatureVerifiable.swift

```diff
@@ -7,24 +7,21 @@ public protocol SignatureVerifiable {
 }
 
 extension SignatureVerifiable {
-
-    @available(*, deprecated, message: "Use `verifySignature(using:)` instead")
-    public func verifySignatureWith(_ signer: Signer) throws -> Bool {
-        return try verifySignature(using: signer)
-    }
-
     /// Verifies signature
     ///
     /// - parameter signer: used to verify the signature
     ///
     /// - throws: JWTError.wrongAlgorithm if the algorithm does not match. Throws any error thrown while signing or encoding.
```

Comment:
Ah good catch. I should have checked the doc blocks. Vapor's other modules don't really specify `- throws` but I think it's a nice touch. I'll go through an update them.